README.md

```diff
@@ -35,18 +40,21 @@ npm install -g ganache-cli@6.4.1
 
 Download and install python 3.6 and its respective pip and virtualenv (**python 3.7 will break things**). Then:
 
+from devtools directory:
+
 ```
 python3 -m venv venv
 source venv/bin/activate
-./devtools/install_python_requirements.sh
+./install_python_requirements.sh
 ```
 
 **Front-End**
 
 Our frontend uses react.
 
+from app directory:
+
 ```
-cd app
```

Comment:
this was too dependent on guessing which directory user was in. Makes more sense to just take `cd` as assumed knowledge and let the user get to the right dir